### DISCUSSION TITLE
Update the built-in libraries and their methods

### DISCUSSION THREAD

**Discussion by wiltonnsz** [Sketchware-Pro/Sketchware-Pro#469] [Ideas]
I don't know if this has already been done for the next version of sketchware pro, but it sure would be nice if the built-in libraries (like Firebase and AppCompat) were updated to a newer version.

**Comment by j-px** [comment]
Note that library names in libs.zip and dexs.zip don't always represent library versions. For example, the Material Design Components library is at v1.5.0-beta01.

**Reply by j-px** [reply]
Update: They now do.

**Comment by AdesonBezerra** [comment]
It has several deprecated methods that could be updated

**Comment by j-px** [comment]
Another note: You can more easily upgrade library versions on your own with the Exclude built-in libraries feature! More on that in #696.
